frontend/src/locales/en.yml

```diff
@@ -576,6 +580,7 @@ instance:
     Bytebase requires read/write (NOT the super privilege) access to the
     instance in order to perform database operations on behalf of the user.
   search-instance-name: Search instance name
+  dbs-ro-unsupport: The {database} does not currently support creating read-only connections
```

Comment:
no-read-only-data-source-support

Be consistent with other parts and avoid using abbreviation